libsql_test.go

```diff
@@ -213,9 +214,9 @@ func (s PreparedStmt) exec(args ...any) sql.Result {
 }
 
 type Tx struct {
-	*sql.Tx
-	t   Table
-	ctx *context.Context
+	ctx   *context.Context
+	rawTx *sql.Tx
+	t     Table
```

Comment:
Why is this useful?

Reply:
The struct size was 72 bytes when ordering it like in new PR it reduced to 64 bytes.

https://go101.org/article/memory-layout.html
https://www.wagslane.dev/posts/go-struct-ordering/

Reply:
It also adds an option to use it as a keyed struct when you embed another struct you cannot use them as keyed.